app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,12 @@
  *= require foundation_and_overrides
 
  */
+
+ body {
+  background: url("/background.png") no-repeat center fixed;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+  background-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.